### PR TITLE
Generation of 16-digit+ numbers as text cells in spreadsheet (Connect #2291)

### PR DESCRIPTION
**Pull request by stellanl**
#### Before the PR (what is the issue or what needed to be done)
16-digit numbers got rounded.
#### The solution
output them as text, and warn the user with a note on the column.
## Checklist
* [X] Connect the issue
* [ ] Test plan
* [ ] Copyright header
* [ ] Code formatting
* [ ] Documentation

### CODE REVIEW

File: GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java [hunk 1]
```diff
@@ -1302,7 +1353,7 @@ protected void createRawDataHeader(Workbook wb,
                     if (qgSheetMap.containsKey(group.getKeyId()))   {
                         Sheet groupSheet = qgSheetMap.get(group.getKeyId());
                         int metaEnd = addMetaDataHeaders(groupSheet, true);
-                        addComment(groupSheet);
+//                        addComment(groupSheet, 0, 0, REPORT_COMMENT);
```
Comment (muloem) [inline GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java]: if its commented out I think lets remove it completely for now.  When we need it again we will add it in.

File: GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java [hunk 2]
```diff
@@ -431,7 +439,8 @@ public int compare(String o1, String o2) {
                 wb.setActiveSheet(isFullReport ? wb.getNumberOfSheets()-1 : 0);
                 wb.write(fileOut);
                 fileOut.close();
-
+                
+                setDocumentProperties(fileName, surveyId);
```
Comment (muloem) [inline GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java]: I think we should also cut this part out. Maybe we can have it in another branch and only if verified that it doesn't cause any performance issues, merge it. For this issue lets only focus on the generating of the text for 16-digit+ numbers.

File: GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java [hunk 3]
```diff
@@ -442,6 +451,38 @@ public int compare(String o1, String o2) {
...
+        POIXMLProperties.CustomProperties custProp = props.getCustomProperties();
+        custProp.addProperty("Version", "Akvo Flow v1.9.25");
+        custProp.addProperty("Generated", new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZ").format(new Date()));
+        
+        FileOutputStream fos = new FileOutputStream(fileName);
+        workbook.write(fos);
+        fos.close();
+    }
+
+    private void addColumnWarnings(Workbook wb) {
```
Comment (muloem) [inline GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java]: As this is unused for now, I think we should also remove till we get round to using it.

File: GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java (hunk 2 above)
Comment (stellanl) [reply GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java]: Ok, I just thought the big change in reporting formats would be a good time to start tagging the files.

**Pull request merged**